app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
```

Comment:
I think we'd need something to make sure URIEnabledStringField happens in our code base (I'm assuming here upstream won't want that, but maybe I'm crazy.)